mentalmodel.md

```diff
@@ -0,0 +1,94 @@
+# Mental Object Model
+
+The temporal proposal works within a specific mental model of objects. These represent very specific concepts and a hierarchy of interaction. The following diagram should illustrate the relationships:
+
+```mermaid
+graph TB
+  instant((Instant))
+    offset((OffsetDateTime))
+    zoned((ZonedDateTime))
+      datetime((CivilDateTime))
+      time((CivilTime))
+      date((CivilDate))
+      yearmonth((CivilYearMonth))
+      monthday((CivilMonthDay))
+      
+  instant --- offset
+  instant --- zoned
+  offset --- datetime
+  zoned --- datetime
+  datetime --- date
+  datetime --- time
+  date --- yearmonth
+  date --- monthday
+```
+![Mental Model Diagram](./objectrelation.png)
+
+Before continuing I'd also like to call out the following API principles to keep in mind when evaluating the proposal:
+
+1. All objects are immutable
+2. Methods beginning with `with` will result in objects that contain equal or more data and are therefore safe in terms of lossing information.
+3. Methods beginning with `get` will result in retrieving an aspect of the object. Thus the value retrieved will have less information that the original object.
+4. The calculation methods `plus`, `minus` and `difference` act with or result in objects of the same type. `difference` is only permissible with objects of the same information content, while `plus` and `minus` are permissible only with `Durations` (or objects containing the same level of information) generated using the `difference` method.
+
+**Please also see the detailed [API description of these temporal objects](objects.md)!!!**
+
+## Instant
+
+`Instant` objects are meant to represent a specific point in absolute time. For convenience we have chosen that point to be *midnight of January the 1st 1970 UTC* as our *epoch*. The choice fell upon that specific date and time due to the precendent of the [POSIX Epoch](http://pubs.opengroup.org/onlinepubs/9699919799/mindex.html) and Unix Epoch based on it.
+
+We decided to ignore  leap seconds since they are arbitrarily decided upon and unpredictable. An `Instant` can therefore not be said to be on the TAE timeline, but rather on the UTC timeline. In terms of leap-seconds we accept them a input (`fromString`) but immediately convert them to the relevant numeric representation and therefore ignoring the fact that this came from a leap-second.
```

Comment:
```suggestion
We decided to ignore leap seconds since they are arbitrarily decided upon and unpredictable. An `Instant` can therefore not be said to be on the TAE timeline, but rather on the UTC timeline. In terms of leap-seconds we accept them a input (`fromString`) but immediately convert them to the relevant numeric representation and therefore ignoring the fact that this came from a leap-second.
```